### PR TITLE
Documentation update for crystal collimators

### PR DESCRIPTION
**Pull request by MarcoDAndrea**
The "Special elements" section of the user manual has been updated with information on crystal collimators, including the description of the additional parameters required by the collimator database as well as appropriate references.

### CODE REVIEW

File: doc/user_manual/chSpecialElements.tex [hunk 1]
```diff
@@ -2035,6 +2035,31 @@ \subsubsection{Additional Collimator Settings}
 JAW_FIT tcsg7 15 FIT_A FIT_B 1.0 1.0 on on
 \end{cverbatim}
 
+\paragraph{Crystal Collimators} \texttt{CRYSTAL name Bend XDim YDim Thick Tilt MisCut Orient}\\
+
+This flag enables the collimator specified in \texttt{name} to be treated as a crystal collimator.
+Crystal collimators require an additional set of parameters described in the following:
+
```
Comment (vkbo) [inline doc/user_manual/chSpecialElements.tex]: can you add `\index{crystal collimator}` at the end of the first paragraph? It adds a reference in the index at the back of the manual.

File: doc/user_manual/chSpecialElements.tex [hunk 2]
```diff
@@ -2035,6 +2035,31 @@ \subsubsection{Additional Collimator Settings}
 JAW_FIT tcsg7 15 FIT_A FIT_B 1.0 1.0 on on
 \end{cverbatim}
 
+\paragraph{Crystal Collimators} \texttt{CRYSTAL name Bend XDim YDim Thick Tilt MisCut Orient}\\
+
+This flag enables the collimator specified in \texttt{name} to be treated as a crystal collimator.
```
Comment (vkbo) [inline doc/user_manual/chSpecialElements.tex]: Change "This flag enables the collimator specified"
To "This flag enables the collimator or collimator family specified"

File: doc/user_manual/theBibliography.tex [hunk 3]
```diff
@@ -277,5 +277,13 @@
...
         Commun. ACM 31, no. 6 (June 1988): 742–751. DOI:10.1145/62959.62969.
     %
+    \bibitem{crystal:1}
+    V.~Previtali, ``Performance Evaluation of a Crystal-enhanced Collimation System for the LHC'',
+    Ph.~D.~thesis, EPFL, Lausanne, Switzerland (2010).
+    %
+    \bibitem{crystal:2}
+    D.~Mirarchi, ``Crystal collimation for LHC'',
+    Ph.~D.~thesis, Imperial College, London, UK (2015).
+    %
```
Comment (amereghe) [inline doc/user_manual/theBibliography.tex]: what about adding [this ref](http://cds.cern.ch/record/2694336?ln=en)?

Moreover, does the imported code take into account the [work by F. Forcher](http://cds.cern.ch/search?ln=en&as=1&m1=a&p1=forcher&f1=author&op1=a&m2=a&p2=&f2=title&op2=a&m3=a&p3=&f3=&action_search=Search&c=CERN+Document+Server&sf=&so=a&rm=&rg=100&sc=1&of=hb)?

**Comment by MarcoDAndrea** [comment]
Thank you @vkbo and @amereghe for your comments. I implemented all your suggestions, added a couple more references as suggested by Daniele and removed the "to be implemented" note for crystals among the collimator families. I also added some comments in the crystal module to clarify the meaning of each process flag.

**Pull request merged**